### ISSUE TITLE
AuthExipredError exception raised without required argument "response"

### ISSUE THREAD

**Issue by mnelson4** [Copyleaks/Python-Plagiarism-Checker#11]
It looks like there's an error in the library code. `copyleaks.py`, on line 101, is raising the exception `AuthExipredError` but is not providing the needed argument (`response`).

Specifically, it seems when my auth token expires, and I call `Copyleaks::submit_file()`, there's an internal error in the CopyLeaks Python API because `AuthExipredError` is being raised without passing in the required parameter `response`. 

Here's the full error I'm getting:
```
Traceback (most recent call last):
  File "/var/data/websites/earlychildhoodeducator.com/env/lib64/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/var/data/websites/earlychildhoodeducator.com/env/lib64/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/var/data/websites/earlychildhoodeducator.com/env/lib64/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/data/websites/earlychildhoodeducator.com/env/lib64/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/var/data/websites/earlychildhoodeducator.com/plagiarism/views.py", line 45, in exam_plagiarism_check
    result = get_copyleaks_api().submit_scan(scan)
  File "/var/data/websites/earlychildhoodeducator.com/plagiarism/copyleaks.py", line 67, in submit_scan
    Copyleaks.submit_file(PRODUCT, self.get_auth_token(), scan.pk_zfilled(), file_submission)  # sending the submission to scanning
  File "/var/data/websites/earlychildhoodeducator.com/env/lib64/python3.6/site-packages/copyleaks/copyleaks.py", line 137, in submit_file
    Copyleaks.__submit(url, auth_token, scan_id, submission)
  File "/var/data/websites/earlychildhoodeducator.com/env/lib64/python3.6/site-packages/copyleaks/copyleaks.py", line 107, in __submit
    Copyleaks.verify_auth_token(auth_token)
  File "/var/data/websites/earlychildhoodeducator.com/env/lib64/python3.6/site-packages/copyleaks/copyleaks.py", line 101, in verify_auth_token
    raise AuthExipredError()  # expired
TypeError: __init__() missing 1 required positional argument: 'response'
```

**Comment by YehonatanB** [comment]
Hey @mnelson4 , 
Thanks for reporting. The issue was fixed and pushed to the github repository. If you are using pypi, please upgrade to version "copyleaks 3.0.4".

**Comment by mnelson4** [comment]
Awesome thanks!
